scripts/push_images.sh

```diff
@@ -2,30 +2,19 @@
 
 set -e
 
+latest_version=$(curl https://rubygems.org/api/v1/versions/decidim/latest.json | grep -o "[0-9][0-9]*.[0-9][0-9]*.[0-9][0-9]*")
+version=${DECIDIM_VERSION:-$latest_version}
+
 if [[ "${CIRCLE_BRANCH}" == "master" ]]; then
   docker login -u "$DOCKER_USER" -p "$DOCKER_PASS"
 
-  docker tag "decidim/decidim:${CIRCLE_SHA1}-deploy" decidim/decidim:latest-deploy
-  docker tag "decidim/decidim:${CIRCLE_SHA1}-test" decidim/decidim:latest-test
-  docker tag "decidim/decidim:${CIRCLE_SHA1}-dev" decidim/decidim:latest-dev
-  docker tag "decidim/decidim:${CIRCLE_SHA1}" decidim/decidim:latest
+  docker push "decidim/decidim:$version-deploy"
+  docker push "decidim/decidim:$version-dev"
+  docker push "decidim/decidim:$version-test"
+  docker push "decidim/decidim:$version"
 
   docker push decidim/decidim:latest-deploy
   docker push decidim/decidim:latest-dev
   docker push decidim/decidim:latest-test
   docker push decidim/decidim:latest
 fi
-
-if [[ "${CIRCLE_BRANCH}" =~ ^[0-9\.]+$ ]]; then
```

Comment:
We should probably keep that conditional when pushing onto branches at least to have some kind of `backporting` mechanism, right? So only `master` and `0.10`, `0.11`... will release new images.

Reply:
That's a good point. I figured since we had not pushing any images since `0.10`, we would hardly care about backports. But since this now will be automatic, I think it's definitely worth keeping! :+1: